src/store.js

```diff
@@ -1,14 +1,11 @@
 import { createBrowserHistory } from 'history'
-import { apiMiddleware } from 'redux-api-middleware';
 import { applyMiddleware, createStore, compose } from 'redux'
 import { routerMiddleware } from 'react-router-redux'
 import thunk from 'redux-thunk'
 import rootReducers from './reducers'
 
 export const history = createBrowserHistory();
 
-const createStoreWithMiddleware = applyMiddleware(apiMiddleware)(createStore);
```

Comment:
why did you remove that line?

Reply:
because "createStoreWithMiddleware" was declared here and this variable was also never used in that file. So unnecessary declaration of variable was causing error.